apps/frontend/src/lib/storage/types/user/userData/index.ts

```diff
@@ -32,6 +38,7 @@ export const UserDataSchema = z.object({
   lannaHalloween: nullToUndefined(LannaHalloweenDataSchema),
   devcon: nullToUndefined(DevconSchema),
   connectionPSISize: nullToUndefined(ConnectionPSISizeSchema),
+  pullPSI: nullToUndefined(PullPSISchema),
```

Comment:
Adding to keep track of when a connection should automatically refresh their PSI results. I don't love the name, probably will change it tomorrow.

Reply:
if this is related to connections refreshing psi results why is it at the userData struct level?